mne/viz/backends/_qt.py

```diff
@@ -435,6 +436,40 @@ def _window_ensure_minimum_sizes(self, sz):
             self._process_events()
             self._process_events()
 
+    def _window_set_theme(self, theme):
+        if theme == 'auto':
+            try:
+                import darkdetect
+            except ModuleNotFoundError:
+                logger.info('For automatic Dark-Mode-Detection '
+                            '"darkdetect" has to be installed!'
+                            'You can install it with'
+                            ' `pip install qdarkdetect`')
+                theme = 'light'
+            else:
+                detected_theme = darkdetect.theme()
+                if detected_theme is not None:
+                    theme = detected_theme.lower()
+                else:
+                    theme = 'light'
+
+        if theme == 'dark':
+            try:
+                import qdarkstyle
+            except ModuleNotFoundError:
+                logger.info('For Dark-Mode "qdarkstyle" has to be installed! '
+                            'You can install it with `pip install qdarkstyle`')
+                stylesheet = None
+            else:
+                stylesheet = qdarkstyle.load_stylesheet()
+        elif isfile(theme):
```

Comment:
I would rather have this be `elif theme != 'light'`. That way someone passing `theme='my_themem.css'` will not just get a light theme by default, they will get a FileNotFoundError because they passed a typo `my_themem.css` instead of `my_theme.css` or whatever.

The only downside is that someone cannot use a file named `'light'` or `'dark'` which seems fine

Reply:
Ok